src/thunderbots/software/ai/navigator/obstacle/velocity_obstacle.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "../shared/constants.h"
```

Comment:
nit: I don't think you need the `../` relative path here. You should be able to start from `shared/`

```suggestion
#include "shared/constants.h"
```

Reply:
ok